### PR TITLE
Recipe for MetaCluster 5.1

### PR DESCRIPTION
**Pull request by keuv-grvl**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/metacluster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: http://i.cs.hku.hk/~alse/MetaCluster/files/{{ name }}5.tar.bz2
+  sha256linux: {{ sha256 }}
```
Comment (bgruening) [inline recipes/metacluster/meta.yaml]: linux?

File: recipes/metacluster/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: http://i.cs.hku.hk/~alse/MetaCluster/files/{{ name }}5.tar.bz2
```
Comment (bgruening) [inline recipes/metacluster/meta.yaml]: Is this the correct linke? It's just called `5`

Comment (keuv-grvl) [inline recipes/metacluster/meta.yaml]: Yes, the URL only states the major version...

File: recipes/metacluster/meta.yaml (hunk 1 above)
Comment (keuv-grvl) [inline recipes/metacluster/meta.yaml]: Bad reuse of an old variable

**Comment by bgruening** [comment]
Thanks @keuv-grvl!

**Pull request merged**